src/client/java/teammates/client/scripts/DataMigrationHelper.java

```diff
@@ -20,4 +22,13 @@ public static boolean isSanitizedHtml(String string) {
         return string.indexOf("&lt;") >= 0 || string.indexOf("&gt;") >= 0 || string.indexOf("&quot;") >= 0
                || string.indexOf("&#x2f;") >= 0 || string.indexOf("&#39;") >= 0 || string.indexOf("&amp;") >= 0;
     }
+
+    /**
+     * Prints the count on system output when count is a multiple of PRINT_CYCLE
+     */
+    public static void printCountRegularly(int count, String itemName) {
+        if (count % PRINT_CYCLE == 0) {
+            System.out.println("On the " + count + "th " + itemName + ".");
+        }
+    }
 }
```

Comment:
Is this class purely for the use of the `DataMigrationForSanitizedDataInAdminEmailAttributes`? If yes, is it possible to put inside the same file?